Analysis/Tasks/PWGHF/HFCandidateCreator3Prong.cxx

```diff
@@ -198,6 +198,15 @@ struct HFCandidateCreator3ProngMC {
         }
       }
 
+      // Ξc± → p± K∓ π±
+      if (flag == 0) {
+        //Printf("Checking Ξc± → p± K∓ π±");
+        auto indexRecXic = RecoDecay::getMatchedMCRec(particlesMC, std::move(arrayDaughters), 4232, array{+kProton, -kKPlus, +kPiPlus}, true, &sign);
```

Comment:
`std::move` must always go only once for the last usage of `arrayDaughters`, so it has to be removed from `auto indexRecLc = `.

Reply:
I removed it from the MCRec call for Lc

Reply:
You don't seem to need to store the return value of `RecoDecay::getMatchedMCRec`, so you can just directly check it in the `if` condition, as done for the D0.
Same for `isMatchedMCGen`.